build.gradle

```diff
@@ -45,11 +45,28 @@ ext {
     avro_version = "1.8.2"
 }
 
+spotless {
+    java {
+        enforceCheck false
+        googleJavaFormat()
+        importOrder('org.candlepin','com','org','net','ch','io','liquibase','java','javax','') // or importOrderFile
```

Comment:
The google format intellij plugin doesn't take care of import order, so you'd have to import their formatter .xml yourself to get the google style import order (which i'm not a huge fan of....just one giant list and i'm pretty sure it starts using asterisks in imports way sooner than i would personally choose to).  This line is totally optional though, and I lifted the order from what was previously defined in our checkstyle config.